addons/voice/org.openhab.voice.kaldi/README.md

```diff
@@ -0,0 +1,10 @@
+# Kaldi TTS
+
+Kaldi is a speech recognition toolkit, freely available under the Apache License.
+
+Kaldi aims to provide software that is flexible and extensible.
+It supports linear transforms, MMI, boosted MMI and MCE discriminative training,
+feature-space discriminative training, and deep neural networks.
```

Comment:
Second line needed here?

Reply:
Hm, here I slightly disagree. Even in English you can build up really long sentences. And dashes and commas are natural "breathing breaks" and therefore a newline might make sense.

Reply:
Fair point here. No one said that one line per sentence is the "always perfect" rule. 🙂